network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/extensions/PilotPointAdderImpl.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.network.store.iidm.impl.extensions;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.extensions.PilotPointAdder;
+import com.powsybl.network.store.iidm.impl.NetworkImpl;
+import com.powsybl.network.store.model.PilotPointAttributes;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Achour Berrahma <achour.berrahma at rte-france.com>
+ */
+public class PilotPointAdderImpl implements PilotPointAdder {
+
+    private final ControlZoneAdderImpl parent;
+
+    private final NetworkImpl network;
```

Comment:
Useless : network is already available in parent (just add a getter in ControlZoneAdderImpl) ?